### PR TITLE
fix typos in the inspect json structs

### PR DESCRIPTION
**Pull request by baude**
Resolves: #633

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/inspect/inspect.go [hunk 1]
```diff
@@ -44,23 +44,23 @@ type HostConfig struct {
 	ShmSize              int64                       `json:"ShmSize"`
 	Runtime              string                      `json:"Runtime"`
 	ConsoleSize          *specs.Box                  `json:"ConsoleSize"`
-	Isolation            string                      `json:"Isolation"` //TODO
-	CPUShares            *uint64                     `json:"CPUSShares"`
+	Isolation            string                      `json:"Isolation"`  //TODO
+	CPUShares            *uint64                     `json:"CpusShares"` //nolint
```
Comment (mheon) [inline pkg/inspect/inspect.go]: Isn't this supposed to be CpuShares?

File: pkg/inspect/inspect.go [hunk 2]
```diff
@@ -44,23 +44,23 @@ type HostConfig struct {
...
 	BlkioDeviceWriteBps  []specs.LinuxThrottleDevice `json:"BlkioDeviceWriteBps"`
 	BlkioDeviceReadIOps  []specs.LinuxThrottleDevice `json:"BlkioDeviceReadIOps"`
 	BlkioDeviceWriteIOps []specs.LinuxThrottleDevice `json:"BlkioDeviceWriteIOps"`
-	CPUPeriod            *uint64                     `json:"CPUPeriod"`
-	CPUQuota             *int64                      `json:"CPUQuota"`
-	CPURealtimePeriod    *uint64                     `json:"CPURealtimePeriod"`
-	CPURealtimeRuntime   *int64                      `json:"CPURealtimeRuntime"`
-	CPUSetCPUs           string                      `json:"CPUSetCPUs"`
-	CPUSetMems           string                      `json:"CPUSetMems"`
+	CPUPeriod            *uint64                     `json:"CpuPeriod"`          //nolint
```
Comment (mheon) [inline pkg/inspect/inspect.go]: Do we have to nolint these given it's only the JSON tag that's not properly named? I think we don't

File: pkg/inspect/inspect.go [hunk 3]
```diff
@@ -44,23 +44,23 @@ type HostConfig struct {
 	ShmSize              int64                       `json:"ShmSize"`
 	Runtime              string                      `json:"Runtime"`
 	ConsoleSize          *specs.Box                  `json:"ConsoleSize"`
-	Isolation            string                      `json:"Isolation"` //TODO
-	CPUShares            *uint64                     `json:"CPUSShares"`
+	Isolation            string                      `json:"Isolation"`  //TODO
```
Comment (mheon) [inline pkg/inspect/inspect.go]: I think this is windows-only. @rhatdan Should we throw this out?

Comment (rhatdan) [reply pkg/inspect/inspect.go]: Yes drop Isolation.

**Comment by mheon** [comment]
LGTM, tests green

**Comment by rhatdan** [comment]
LGTM
@rh-atomic-bot r+

**Pull request closed without merging**